src/components/verification-grid/verification-grid.ts

```diff
@@ -1082,18 +1069,31 @@ export class VerificationGridComponent extends AbstractComponent(LitElement) {
 
   //#region Rendering
 
-  private async renderVirtualPage(nextPage: SubjectWrapper[]): Promise<void> {
+  private async renderVirtualPageIndex(index: number): Promise<void> {
+    const virtualPage = this.subjectBuffer.slice(index, index + this.populatedTileCount);
+    await this.renderVirtualPage(virtualPage);
+  }
+
+  private async renderVirtualPage(pageSubjects: SubjectWrapper[]): Promise<void> {
```

Comment:
do we need the flexibility of these overloads?

```suggestion
  private async renderVirtualPage(): Promise<void> {
```

Where the method always just renders from viewHead to populatedTileCount.

I also don't love the slice of the virtual page here. Seems unnecessary.